packages/glob/__tests__/internal-pattern.test.ts

```diff
@@ -1,17 +1,9 @@
 import * as io from '../../io/src/io'
+import * as os from 'os'
```

Comment:
all the changes to this file are reaction due to moving `"esModuleInterop": true` to the root `tsconfig.json` - which previously was only defined in `packages/github/tsconfig.json`

i left a comment on internal-pattern.ts (further below) which explains more